flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -115,18 +115,18 @@ void UniqueKey(List<SqlNodeList> list) :
     }
 }
 
-SqlNode PropertyValue() :
+SqlNode TableOption() :
 {
-    SqlIdentifier key;
+    SqlNode key;
```

Comment:
Force key and value to be `StringLiteral`?

Reply:
The `StringLiteral()`'s return type is `SalNode`, i suggest to keep the type as it is(do not force type cast in parser), we have a type cast in `TableOption`.